masonry/doc/ARCHITECTURE.md

```diff
@@ -20,31 +22,31 @@ Masonry has some opinionated design goals:
 
 ## Code layout
 
-### `src/platform/`
-
-Some platform-specific stuff, for interfacing with Glazier. Relatively empty.
-
 ### `src/testing/`
 
 Contains the TestHarness type, various helper widgets for writing tests, and the snapshot testing code.
 
+### `src/text2/`
+
+Contains text-handling code, for both displaying and editing text. Has been overhauled during the port to winit, but still in rough shape. Here be dragons.
+
 ### `src/text/`
 
-Contains text-handling code, for both displaying and editing text. Hasn't been maintained in a while, here be dragons.
+Dead code, should probably be cleaned up.
```

Comment:
Yeah - it's provided for reference of future direction, e.g. `movement.rs` which we expect to move to Parley.
At this point, we have most of the functionality elsewhere.